PlancakeEmailParser.php

```diff
@@ -412,16 +412,14 @@ public function getBody($returnType = self::PLAINTEXT)
         }
 
         // there could be more than one boundary
-        preg_match_all('!boundary=(.*?)[;$]!mi', $this->emailRawContent, $matches);
+        preg_match_all('/boundary=(?:|")([a-zA-Z0-9\(\)_\/+-]+)(?:|")(?:$|;)/mi', $this->emailRawContent, $matches);
```

Comment:
The [;$] part didn't seem to work on any of the test cases (at least in my php7 installation)